web-ng/src/app/components/map/map.component.ts

```diff
@@ -14,22 +14,63 @@
  * limitations under the License.
  */
 
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { HttpParams } from '@angular/common/http';
 import { Feature } from '../../shared/models/feature.model';
 import { FeatureService } from '../../services/feature/feature.service';
-import { Observable } from 'rxjs';
+import { Observable, Subscription } from 'rxjs';
 import { List } from 'immutable';
+import { Router, ActivatedRoute, NavigationExtras } from '@angular/router';
 
 @Component({
   selector: 'ground-map',
   templateUrl: './map.component.html',
-  styleUrls: ['./map.css'],
+  styleUrls: ['./map.component.css'],
 })
-export class MapComponent {
+export class MapComponent implements OnInit {
+  subscription: Subscription = new Subscription();
   zoom = 3;
+  focusedFeatureId = '';
   features$: Observable<List<Feature>>;
+  icon = {
+    url: 'http://maps.google.com/mapfiles/kml/paddle/red-circle.png',
```

Comment:
Can we import these icons and serve them from our app instead of from google.com?